database/rrdhost.c

```diff
@@ -707,14 +707,27 @@ void rrdhost_save_charts(RRDHOST *host) {
     rrdhost_unlock(host);
 }
 
+int is_valid_label(char *key) {
```

Comment:
It would be nice to get this function in the branch soon, so I can call it too.

Reply:
I changed everything requested, for me we can merge this PR and I create another to finalize #7410, I was also blocked before you merge a PR today.